scripts/astrokat-observe.py

```diff
@@ -6,8 +6,7 @@
 import ephem
 
 import numpy as np
-from itertools import chain
-import time
+
 from datetime import datetime, timedelta
```

Comment:
There are a couple of complaints on this file about indents and unused imports/variables. Maybe hit it with your linter again.

Reply:
astrokat-observe.py has a number of conflicts, this is one of them, I am resolving this out now

Reply:
unused packages removed